ipfsapi/client.py

```diff
@@ -743,6 +743,42 @@ def resolve(self, name, recursive=False, **kwargs):
         args = (name,)
         return self._client.request('/resolve', args, decoder='json', **kwargs)
 
+    def key_gen(self, keyName, type='rsa', size='2048', **kwargs):
```

Comment:
Please make `type` a mandatory argument (we don't want to hard-code a crypto type that may eventually become obsolete) and `size` an *int*.